manifests/b/BelgianGovernment/eIDVieuwer/5.1.2.5886/BelgianGovernment.eIDVieuwer.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BelgianGovernment.eIDVieuwer
+PackageVersion: 5.1.2.5886

```

```suggestion
PackageVersion: 5.1.5886
```
This must match Control Panel to prevent upgrade loops due to version mismatch.